xbmc/interfaces/swig/AddonModuleXbmcgui.i

```diff
@@ -43,6 +43,8 @@ using namespace xbmcgui;
 // not part of what swig parses.
 %feature("knownbasetypes") XBMCAddon::xbmcgui "AddonClass,AddonCallback"
 
+%feature("knownapitypes") XBMCAddon::xbmcgui "XBMCAddon::xbmc::InfoTagVideo,xbmc::InfoTagMusic"
```

Comment:
Did you leave the `XBMCAddo::` namespace for `xbmc::InfoTagMusic` out on purpose?